jobs/openings.html

```diff
@@ -49,7 +49,7 @@ <h4>Monthly Innovation & Technology day</h4>
     <h2>Openings</h2>
     <div class="features">
         {% for job in site.jobs %}
-        <div class="feature">
+        <div class="feature-jobs">
             <i class="fa fa-code"></i>
```

Comment:
I think we'll need the icon to be configurable, because the code icon does not fit anymore with an office assistant. unfortunately I have not found a good "office" or "assistant" icon in the free font awesome icon set. I think we'll buy the full version. do you have an idea which icon we could use instead of the code icon?